tests/functional/test_completion.py

```diff
@@ -251,10 +251,7 @@ def test_completion_files_after_option(
     ), "autocomplete function could not complete <dir> after options in command"
     assert not any(
         out in res.stdout for out in (os.path.join("REPLAY", ""), "README.txt")
-    ), (
-        "autocomplete function completed <file> or <dir> that "
-        "should not be completed"
-    )
+    ), "autocomplete function completed <file> or <dir> that should not be completed"
```

Comment:
... and even more so here. I much prefer the split and parenthesised version. But whatever, I guess.